cairo/ethereum/utils/numeric.cairo

```diff
@@ -148,3 +151,17 @@ func U256__eq__(a: U256, b: U256) -> bool {
     tempvar res = bool(0);
     return res;
 }
+
+func U256_be_from_address{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(address: Bytes20) -> U256 {
```

Comment:
```suggestion
func U256_from_address{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(address: Bytes20) -> U256 {
```

we shouldn't have any endianess info here, its included in Address type. It would be relevant if it was `U256_from_be_bytes20` for example